ui/components/data-sources/DataSourcesContainer.js

```diff
@@ -35,6 +35,12 @@ class DataSourcesContainer extends Component {
         this.setState({ filter: { name } });
     }
 
+    handleKeyPress(event) {
+        if (event.key === "Enter") {
```

Comment:
Very minor, but maybe use `keyCode` instead of the string value.

Reply:
https://stackoverflow.com/a/34143412